pkg/crc/api/api_test.go

```diff
@@ -164,7 +164,7 @@ func TestGetconfigApi(t *testing.T) {
 	assert.NoError(t, json.Unmarshal(payload[:n], &getconfigRes))
 
 	configs := make(map[string]interface{})
-	configs["cpus"] = "5"
+	configs["cpus"] = 5.0
```

Comment:
This is because in this assertion we parse the JSON to map[string]interface{} and Go doesn't know the final type. It prefers to use float64 because it's a "number".